### ISSUE TITLE
Use terminal for all System.out output

### ISSUE THREAD

**Issue by amxrmxhdx** [mabe02/lanterna#448]
Hey! How can i use the Terminal for all System.out output?

**Comment by mabe02** [comment]
Sorry, I'm not sure what you mean here? What are you trying to do?

**Comment by amxrmxhdx** [comment]
I'm trying to use the terminal as an actual terminal, so as System.in and as System.out. Is this possible?

**Comment by avl42** [comment]
See #449 (my answer there also seems to fit this question.)

**Comment by avl42** [comment]
Maybe it is easier if you consider lanterna's Stream/Ansi/Unix Terminal classes as "device drivers" for actual terminals (aka "tty"s), and the graphical AWT/Swing Terminal classes as "dummy drivers" that present the data themselves rather than driving a device.

**Comment by mabe02** [comment]
Yeah, there isn't anything built for this out of the box unfortunately. While we could decode ANSI escape characters and implement a subset of terminal features so you could run a shell with lanterna, there are several terminal characteristics that are done by the OS/devices which we can't emulate from Java (without native bindings). For example, the echo mode, canonical input, screen size change trigger and other things are done using the (pseudo-)terminal device and signals, which we can't do.
